WORKSPACE

```diff
@@ -23,3 +23,17 @@ http_archive(
 load("@google_bazel_common//:workspace_defs.bzl", "google_common_workspace_rules")
 
 google_common_workspace_rules()
+
```

Comment:
what's this in aid of?
I'm not big on Maven stuff, but it looks odd to add this here when none of the other backends did anything like this.

Reply:
log4j2 is not in google_common_workspace_rules so we need to put it here so that flogger can be built.